package.sh

```diff
@@ -1,49 +1,25 @@
 #!/bin/bash -e
 
-# Thanks to the zigbee-addon https://github.com/mozilla-iot/zigbee-adapter
-# from Mozilla IoT  for parts of this script!
-
-# Architecture options:
-#    darwin-x64
-#    linux-arm
-#    linux-arm64
-#    linux-ia32
-#    linux-ia64
-#    win32-ia32
-#    win32-x64
-
-ADDON_ARCH=$1
-HOST_ARCH=`uname -s`
-
-if [ -z $ADDON_ARCH ]; then
-   echo Please specify an architecture.
-   exit 1
-fi
-
-NODE_VERSION="$(node --version)"
-TARFILE_SUFFIX="-${ADDON_ARCH}-${NODE_VERSION/\.*/}"
-
-if [ $HOST_ARCH == "Darwin" ]; then
-    SHASUM='shasum -a 256'
+rm -rf node_modules
+if [ -z "${ADDON_ARCH}" ]; then
+  TARFILE_SUFFIX=
 else
-    SHASUM='sha256sum'
+  NODE_VERSION="$(node --version)"
+  TARFILE_SUFFIX="-${ADDON_ARCH}-${NODE_VERSION/\.*/}"
 fi
-
-rm -rf node_modules
-
 # For openwrt-linux-arm and linux-arm we need to cross compile.
 if [[ "${ADDON_ARCH}" =~ "linux-arm" ]]; then
-    # We assume that CC and CXX are pointing to the cross compilers
-    npm install --ignore-scripts --production
-    npm rebuild --arch=armv6l --target_arch=arm
+  # We assume that CC and CXX are pointing to the cross compilers
+  npm install --ignore-scripts --production
+  npm rebuild --arch=armv6l --target_arch=arm
 else
-    npm install --production
+  npm install --production
 fi
 
 rm -f SHA256SUMS
-${SHASUM} manifest.json package.json *.js LICENSE README.md > SHA256SUMS
+sha256sum manifest.json package.json *.js LICENSE README.md > SHA256SUMS
```

Comment:
sha256sum is not a valid command in MacOS X.  Apparently, it's only available as a homebrew install of coreutils.  If this will only ever by built by the build server, then this is fine.  Otherwise, the host architecture selection may need to be added back.